CHANGELOG.md

```diff
@@ -1,5 +1,315 @@
 # NHS.UK frontend Changelog
 
+## 6.0.0 - 22 November 2021
```

Comment:
Will need to change the date to the actual release date. Do we normally put TBC?

Reply:
Yeah this release will most likely be going out Monday now